ntpclient/ntpclient_test.go

```diff
@@ -1,25 +1,16 @@
 package ntpclient
 
 import (
+	"reflect"
 	"testing"
+	"time"
 )
 
 func TestNTPClient(t *testing.T) {
 	pool := []string{"pool.ntp.org:123", "0.pool.ntp.org:123"}
-	_, err := NTPClient(pool)
-	if err != nil {
-		t.Fatalf("failed to get time %v", err)
-	}
-
-	invalidpool := []string{"pool.thisisinvalid.org"}
-	_, err = NTPClient(invalidpool)
-	if err == nil {
-		t.Errorf("Expected error")
-	}
+	v := NTPClient(pool)
 
-	firstInvalid := []string{"pool.thisisinvalid.org", "pool.ntp.org:123", "0.pool.ntp.org:123"}
-	_, err = NTPClient(firstInvalid)
-	if err != nil {
-		t.Errorf("failed to get time %v", err)
+	if reflect.TypeOf(v) != reflect.TypeOf(time.Time{}) {
```

Comment:
This will always be a time.Time so can RM this check. Or could force a time.Now().UTC() check?

Reply:
Honestly this was just a lazy way to maintain test coverage to at least know its returning a time.Now()

I can do a time.Now.UTC() check but i would have to also parse the time into a format that doen't include anything above milliseconds or return value of NTPClient and time.Now.UTC() will be out and cause a failure

Reply:
Another one is just comparing the year/day/month is the same, that's a good enough check

Reply:
I kept the type check because you can never have to much test coverage right :D?
Also added a check for IsZero() and using 2006-01-02T15:04:05 format